src/Workleap.DomainEventPropagation.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
-#nullable enable
\ No newline at end of file
+#nullable enable
+*REMOVED*Workleap.DomainEventPropagation.IDomainEvent.DataVersion.get -> string!
```

Comment:
I think we can leave it empty since we haven't released any real version yet

Reply:
The card say remove and harcode to 1.0

Reply:
Oh yeah yeah, I mean this additional line in Unshipped that says the property has been removed